settings.gradle.kts

```diff
@@ -46,5 +47,8 @@ dependencyResolutionManagement {
 
             alias("guava").to("com.google.guava:guava:31.0.1-jre")
         }
+        create("testcontainers") {
+            version("mongo", "5.0.5-focal")
```

Comment:
Centralized version management: created a separate catalogue for [TestContainers](https://www.testcontainers.org/) images.